README.md

```diff
@@ -112,6 +112,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | PingCap | Required for Bay Area | Restricted | Restricted | Restricted | 2020-03-07 |
 | Pinterest | Encouraged | Restricted | Restricted | Partially Restricted | 2020-03-08 |
 | Polidea | Encouraged | Restricted | Restricted | Restricted | 2020-03-10 |
+| Positronic AI | Required | Restricted | Restricted | Restricted | 2020-03-11 |
```

Comment:
Do you have a public-facing post for this change by chance?

Also, would you mind updating the count?

Reply:
Rebased from most current master. Updated count. Official announcement was over (internal) Slack yesterday.
![image](https://user-images.githubusercontent.com/130908/76486217-b77bd880-63ec-11ea-9b12-5517c3ed8b94.png)

Will a tweet do? https://twitter.com/positronicAI/status/1237953785999306752

Reply:
The tweet is great!